src-go/server/.gitignore

```diff
@@ -1,2 +1,3 @@
 /.idea/
-*.der
\ No newline at end of file
+*.der
+/src/main/java/burp/.idea/
```

Comment:
This shouldn't be needed? `.idea` is already excluded above.

Reply:
I added it in the wrong gitignore, it's for the intellij idea project, not for Go. Moved this path to the root gitignore.